testscript/cmd.go

```diff
@@ -141,6 +141,12 @@ func (ts *TestScript) doCmdCmp(args []string, env bool) {
 		// update the script.
 	}
 
+	const maxLineDiff = 1_000_000
```

Comment:
Document how you arrived at this number? For example:

```
pkg/diff is quadratic at the moment.
If the product of the number of lines in the inputs is too large,
don't call pkg.Diff at all as it might take tons of memory or time.
We found one million to be reasonable for an average laptop.
```

Reply:
Good point. I was being lazy given this is a temporary change. Done.